src/coreclr/jit/lsra.cpp

```diff
@@ -297,9 +297,10 @@ regMaskTP LinearScan::getMatchingConstants(regMaskTP mask, Interval* currentInte
     regMaskTP result     = RBM_NONE;
     while (candidates != RBM_NONE)
     {
-        regMaskTP candidateBit = genFindLowestBit(candidates);
-        candidates &= ~candidateBit;
-        regNumber  regNum        = genRegNumFromMask(candidateBit);
+        regNumber regNum       = genFirstRegNumFromMask(candidates);
+        regMaskTP candidateBit = genRegMask(regNum);
+        candidates ^= candidateBit;
```

Comment:
Can this one not be?
```suggestion
        regNumber regNum = genFirstRegNumFromMaskAndToggle(candidates);
```

Reply:
no, because we need `candidateBit` few lines below. That's why I cannot use `genFirstRegNumFromMaskAndToggle()`.